apps/vr-tests-react-components/src/utilities/TestWrapperDecorator.tsx

```diff
@@ -61,37 +53,23 @@ const useNoAnimationStyles = makeStyles({
     },
   },
 });
-export const TestWrapperDecoratorNoAnimation: DecoratorFunction<ExtendedStoryFnReturnType> = story => {
+export const TestWrapperDecoratorPauseAnimation: DecoratorFunction<ExtendedStoryFnReturnType> = story => {
   const noAnimationStyles = useNoAnimationStyles();
-  const className = mergeClasses(noAnimationStyles.root, 'testWrapper');
+  const className = mergeClasses(noAnimationStyles.paused, 'testWrapper');
   return (
     <div style={{ display: 'flex' }}>
       <div
         className={className}
         style={{
           padding: '10px',
-          overflow: 'hidden',
         }}
       >
         {story()}
       </div>
     </div>
   );
 };
```

Comment:
We should not need this if we have `PauseAnimationDecorator`. The point of `PauseAnimationDecorator` is to combine it with one of the existing TestWrapperDecorators.

```
  .addDecorator(PauseAnimationDecorator)
  .addDecorator(TestWrapperDecoratorFullWidth)
```